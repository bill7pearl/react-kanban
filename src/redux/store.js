import { configureStore, applyMiddleware } from '@reduxjs/toolkit';
import thunk from 'redux-thunk';
import rocketReducer from './rockets/rockets';
import missionReducer from './missions/missions';

const store = configureStore(
  {
    reducer: {
      rockets: rocketReducer,
      missions: missionReducer,
    },
  },
  applyMiddleware(thunk),
);

export default store;
