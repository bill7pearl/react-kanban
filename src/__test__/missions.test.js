import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import RenderMissions from '../components/missions';
import store from '../redux/configureStore';

const App = () => {
  <Provider store={store}>
    <RenderMissions />
  </Provider>;
};

describe('Display missions component test', () => {
  it('Tests missions page rendering and fetching data from API', () => {
    const MissionsRenderer = renderer.create(<App />).toJSON();
    expect(MissionsRenderer).toMatchSnapshot();
  });
});
