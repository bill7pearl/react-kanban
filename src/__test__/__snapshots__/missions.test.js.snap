// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display missions component test Tests missions page rendering and fetching data from API 1`] = `null`;
